qml/pages/ImagePage.qml

```diff
@@ -43,6 +44,8 @@ Page {
     property real oldCenterX;
     property real oldCenterY;
 
+    property bool imageOnly
```

Comment:
I could eliminate unnecessary semicolons in other places for consistency ))

Reply:
No need to tease me all the time...